src/App.jsx

```diff
@@ -14,45 +16,103 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_ALPHABETICALLY = 'Sort alphabetically';
+const SORT_LENGTH = 'Sort by length';
+
+export const App = () => {
+  const [chosenFilter, setChosenFilter] = useState('');
+  const [isReverse, setIsReverse] = useState(false);
+  let sortedGoods = [...goodsFromServer];
+
+  if (chosenFilter) {
+    switch (chosenFilter) {
+      case SORT_ALPHABETICALLY:
+        sortedGoods = sortedGoods.sort((good1, good2) => {
+          if (isReverse) {
```

Comment:
don't need to duplicated reverse login in every case, you can do it once after switch case block